tariff/fixed/day.go

```diff
@@ -0,0 +1,116 @@
+package fixed
+
+import (
+	"errors"
+	"fmt"
+	"strconv"
+	"strings"
+
+	"golang.org/x/exp/slices"
+)
+
+//go:generate enumer -type Day
+type Day int
+
+const (
+	Sunday Day = iota
+	Monday
+	Tuesday
+	Wednesday
+	Thursday
+	Friday
+	Saturday
+)
+
+var Week = []Day{Monday, Tuesday, Wednesday, Thursday, Friday, Saturday, Sunday}
+
+var shortDays = map[string]Day{
+	// english
+	"sun": Sunday,
+	"mon": Monday,
+	"tue": Tuesday,
+	"wed": Wednesday,
+	"thu": Thursday,
+	"fri": Friday,
+	"sat": Saturday,
+	// german
+	"so": Sunday,
+	"mo": Monday,
+	"di": Tuesday,
+	"mi": Wednesday,
+	"do": Thursday,
+	"fr": Friday,
+	"sa": Saturday,
```

Comment:
Vorschlag: Auf Englisch beschränken, da wir sonst auch später weiter Sprachen reinnehmen müssten. Solange das per Hand in der Config stattfindet halte ich Englisch für total ok, wenn wir das in die UI ziehen haben wir sowieso i18n am Start.